app/services/text_handler.rb

```diff
@@ -22,8 +22,25 @@ def process_input(user, params)
         user.save
         I18n.t('texts.stop_success_response')
       end
+    else
+      process_rsvp(user, body)
+    end
+  end
+
+  def process_rsvp(user, body)
+    event = Event.find_by_id(body[0...-1])
```

Comment:
I believe the find method just defaults to using ID. What is this [0...-1] doing?

Reply:
Gets all but last char, the last char being Y/N (the attendance case)